docs/faq.rst

```diff
@@ -19,7 +19,9 @@ ReproZip can trace a database server; however, because of the format it uses to
     additional_patterns:
       - /var/lib/mysql/**
 
-Also note that ReproZip does not currently save the state of the files. Therefore, if your experiment modifies a database, ReproZip will pack the already modified data (not the one before tracing the experiment execution).
+See :ref:`Capturing Connections to Servers <packing-clientserv>` for an example using a database and additional information.
```

Comment:
Yes, feel free to add "local" there.

On Friday, May 20, 2016, Remi Rampin <notifications@github.com> wrote:

> In docs/faq.rst
> <https://github.com/ViDA-NYU/reprozip/pull/187#discussion_r64088867>:
>
> > @@ -19,7 +19,9 @@ ReproZip can trace a database server; however, because of the format it uses to
> >      additional_patterns:
> >        - /var/lib/mysql/**
> >
> > -Also note that ReproZip does not currently save the state of the files. Therefore, if your experiment modifies a database, ReproZip will pack the already modified data (not the one before tracing the experiment execution).
> > +See :ref:`Capturing Connections to Servers <packing-clientserv>` for an example using a database and additional information.
>
> using a *+local* database, maybe?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ViDA-NYU/reprozip/pull/187/files/5ee851567ecaf23af856b96c164aba781371642a#r64088867>
>


-- 
Sent from Gmail Mobile